core/pom.xml

```diff
@@ -385,7 +385,7 @@
 		<dependency>
 			<groupId>com.github.mwiede</groupId>
 			<artifactId>jsch</artifactId>
-			<version>0.2.14</version>
+			<version>0.2.15</version>
```

Comment:
Oh, durf je dit hier toch wel aan? Prima op zich. Had het teruggedraaid in mijn PRs.